app/components/ui/textarea.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+import { cn } from '~/lib/utils'
+
+export interface TextareaProps
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+const Textarea = React.forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ className, ...props }, ref) => {
+    return (
+      <textarea
```

Comment:
**praise**: The use of the `cn` utility function for class names is a nice touch for maintaining clean and readable code.

Reply:
this is a shadcn/ui component